modular_zubbers/code/modules/clothing/neck_wearing.dm

```diff
@@ -31,6 +31,7 @@ Use CTRL + SHIFT + LEFT CLICK to turn them on and off.
 	if(!iscarbon(user))
 		return NONE
 	if(only_functional)
+		to_chat(user, span_warning("You can't swap [src] to non-functional mode!"))
```

Comment:
```suggestion
		to_chat(user, span_danger("[src] does not have a non-functional mode!"))
```
makes it more clear imo that this is an intentional block, and not failing to swap due to external factors